examples/cancel_multiple_notifications.py

```diff
@@ -0,0 +1,52 @@
+import time
+from pathlib import Path
+
+from mac_notifications import client
+from mac_notifications.client import Notification
+
+
+if __name__ == "__main__":
+    print("Sending meeting notification.")
+
+    sent: [Notification] = [] 
```

Comment:
I don't think this is correct Python3.8 syntax.

```suggestion
    sent: List[Notification] = [] 
```

Reply:
We also need to add the `from typing import List`

Reply:
Fixed with `list` builtin